.circleci/config.yml

```diff
@@ -0,0 +1,59 @@
+version: 2.1
+
+references:
+  images:
+    go: &GOLANG_IMAGE circleci/golang:latest
```

Comment:
For the products, we normally will pin it to a specific version so we don't have to dig through and compare when builds break to find out that `latest` was a version bump that caused issues. In this case if it works for the latest one, you can pick that one and set it and we can just manually bump it when needed in the future.